cmd/controller/cmd/root.go

```diff
@@ -97,12 +97,14 @@ func init() {
 	rootCmd.AddCommand(viper.GetConfigCommand())
 }
 
-func initConfig(_ *cobra.Command, _ []string) error {
+func initConfig(cmd *cobra.Command, _ []string) error {
 	configAccessor = viper.NewAccessor(config.Options{
 		StrictMode:  true,
 		SearchPaths: []string{cfgFile},
 	})
 
+	configAccessor.InitializePflags(cmd.Flags())
```

Comment:
isnt this already initialized - https://github.com/lyft/flytectl/blob/master/cmd/get/task.go#L62

Reply:
it is a different issue

Reply:
@kumare3 any update on this pr